be/src/exec/es/es_scan_reader.cpp

```diff
@@ -62,7 +62,10 @@ ESScanReader::ESScanReader(const std::string& target, const std::map<std::string
     } else {
         _exactly_once = false;
         // scroll request for scanning 
-        _init_scroll_url = _target + REQUEST_SEPARATOR + _index + REQUEST_SEPARATOR + _type + "/_search?scroll=" + _scroll_keep_alive + REQUEST_PREFERENCE_PREFIX + _shards + "&" + REUQEST_SCROLL_FILTER_PATH;
+        // add terminate_after for the first scroll to avoid decompress all postings list
+        _init_scroll_url = _target + REQUEST_SEPARATOR + _index + REQUEST_SEPARATOR + _type + "/_search?scroll=" + _scroll_keep_alive + REQUEST_PREFERENCE_PREFIX + _shards
+                                   + "&" + REUQEST_SCROLL_FILTER_PATH
+                                   + "&terminate_after=" + batch_size_str;
```

Comment:
Better to use Substitute or std::stringstream
There are too many string add operation, it will introduce too many string copy